src/content/docs/tips/tailwind-v4.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Tailwind v4
+description: Quick set up guide for Tailwind v4
+astroRange: ^4.0.0
+---
+import { Tabs, TabItem, Steps } from "@astrojs/starlight/components";
+
+How to use Tailwind's v4 Vite plugin instead of Astro's Tailwind integration.
+
+Be sure to check out the [Tailwind v4 Blogpost](https://tailwindcss.com/blog/tailwindcss-v4-alpha) for details on using v4.
+
```

Comment:
Can you add a caution or note that states that v4 is still in alpha? so people don't complain to us if something does not work